apps/api-extractor/src/api/Extractor.ts

```diff
@@ -391,6 +392,8 @@ export class Extractor {
 
     collector.analyze();
 
+    VisibilityChecker.check(collector);
```

Comment:
This pattern is odd to me. It's like this is a plugin, but its applied by the the application directly.

Reply:
I wanted the visibility checking logic to be in its own source file, but currently it doesn't maintain any state.  That's why I used a static class.